crates/router/src/connector/cybersource/transformers.rs

```diff
@@ -231,12 +236,22 @@ pub struct ProcessingInformation {
     action_list: Option<Vec<CybersourceActionsList>>,
     action_token_types: Option<Vec<CybersourceActionsTokenType>>,
     authorization_options: Option<CybersourceAuthorizationOptions>,
-    commerce_indicator: CybersourceCommerceIndicator,
+    commerce_indicator: String,
```

Comment:
Why are we changing this from Enum to String?

Reply:
We need to consume the response from cybersource and pass it in 3ds authorise call, we cannot hard code it to internet in this case